mapdata/WarcraftLegacies/AbilityData/Skin/1093808193.json

```diff
@@ -73,25 +73,25 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "The hero empowers nearby allies of nature increasing their armor by 5 and attack by 20% for 30 seconds."
+      "Value": "The hero empowers nearby allies of nature increasing their armor by 5 and attack by 10% for 30 seconds."
```

Comment:
We should be using data fields for all of this stuff! Otherwise we're just going to run into the exact same problem later and it's a waste of time. Not your fault it was incorrect though.